test/other_tests/twoD_vs_oneD.cpp

```diff
@@ -126,6 +126,8 @@ void OneD::set(uint8_t row, uint8_t col, uint8_t val) {
 
 ////////////////////
 
+/* TODO: comment out because can't have two main(); think of a way to let all tests works.
```

Comment:
@Samarium150 this

Reply:
should we care, or just manually change main() entrance every time

Reply:
> should we care, or just manually change main() entrance every time

we can create multiple executables